From 5d35b1a121afcbf3910e2ef9f68617bb4f14c653 Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Fri, 8 Jun 2018 10:18:39 +0800
Subject: [PATCH 1574/2345] kernel/drm/i915/gvt: use highpri workqueue for
 vblank notify

Use highpri workqueue to inject vblank msi in time
to avoid vblank time out in guest.

Signed-off-by: Zhipeng Gong <zhipeng.gong@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index ab9449a..5ead4b7 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -1734,7 +1734,7 @@ static inline void gvt_notify_vblank(struct drm_i915_private *dev_priv,
 				     enum pipe pipe)
 {
 	if (dev_priv->gvt)
-		queue_work(system_unbound_wq,
+		queue_work(system_highpri_wq,
 				&dev_priv->gvt->pipe_info[pipe].vblank_work);
 }
 #endif
-- 
2.7.4

